docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>Same than for ephys, can you choose a meaningful time for the start of the plot. It is easier to grab how useful the plot if if there is a stimulus evoked response on the plot. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>